genetics.pde

```diff
@@ -125,7 +129,7 @@ void tickPopulation() {
   }
   
   for (int[] baby : babies) {
-    population.add(new Critter(baby[0], baby[1], 5));
+    population.add(new Critter(baby[0], baby[1], critterSize/2)); //changed baby size to critter size / 2
```

Comment:
I would use the sizeless constructor here
`population.add(new Critter(baby[0], baby[1]));`

Reply:
Doesn't look like there is a sizeless constructor. I'm just going to hard code in CRITTER_SIZE. I think it'd be nice to inherit size, but I think I'd like to check that with the club before implementing.

Reply:
I misread the code - there is a sizeless constructor but you can't pass it a coordinate.